src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/TypeBuilderImpl.cs

```diff
@@ -439,6 +439,7 @@ protected override FieldBuilder DefineUninitializedDataCore(string name, int siz
             return DefineDataHelper(name, new byte[size], size, attributes);
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2072:DynamicallyAccessedMembers", Justification = "Types are preserved via ModuleBuilderImpl.s_coreTypes")]
```

Comment:
This warning is complaining about members not being preserved. `ModuleBuilderImpl.s_coreTypes` does not preserve the members, so this justification is not quite right.

Reply:
I do not think there is anything real to fix, except the text of the justification.

Reply:
No any member of `ValueType` is referenced in the method, how about updating the message to `ValueType members are not referenced in this context`?

Reply:
LGTM